.circleci/config.yml

```diff
@@ -0,0 +1,114 @@
+version: 2.0
+
+install: &install
+  name: Install bundle
+  command: bundle install --path vendor/bundle
+
+missingdeps: &missingdeps
+  name: Install missing dependecies
+  command: |
+            cat /etc/os-release
+            printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /tmp/sources.list
+            sudo cp /tmp/sources.list /etc/apt/sources.list
+            sudo apt-get update
+            sudo apt-get install cmake libicu-dev libssl-dev
+  
+test: &test
+  name: Test bundle
+  command: bundle exec rake test
+  
+jobs:
+  "ruby-2.1.10":
+    docker:
+      - image: circleci/ruby:2.1.10
+    working_directory: ~/fluent-plugin-kubernetes_metadata_filter
+    steps:
+      - run: *missingdeps
+      - checkout
+      - run: *install
+      - run: *test
+      - store_test_results:
+            path: coverage
+      - store_artifacts:
+            path: coverage
+
+  "ruby-2.2.10":
+    docker:
+      - image: circleci/ruby:2.2.10
+    working_directory: ~/fluent-plugin-kubernetes_metadata_filter
+    steps:
+      - run: *missingdeps
+      - checkout
+      - run: *install
+      - run: *test
+      - store_test_results:
+            path: coverage
+      - store_artifacts:
+            path: coverage
+
+  "ruby-2.3.8":
```

Comment:
Do we really need to test ruby 2.3 and earlier?  What major platforms still require ruby 2.3 and earlier?

Reply:
It was in the  circleci 1.0 config file, just migrated them to the 2.0 config.
We will be more than happy throw old ruby versions away.